tools/MUXPGODatabase/config.ps1

```diff
@@ -0,0 +1,7 @@
+$pgoBranch           = "master"
+$packageId           = "MUXPGODatabase-test"
+
+# Get release version
```

Comment:
Can you add a readme.md to this directory that outlines how this system works? It would be useful to keep that as a record.